src/main/java/io/antmedia/rest/RestServiceBase.java

```diff
@@ -1022,9 +1022,6 @@ public boolean checkStreamUrl (String url) {
 				ipAddr = ipAddrParts[0];
 
 			}
-			if(ipAddr.split("\\.").length == 4 && !validateIPaddress(ipAddr)){
```

Comment:
If it's an IP address and it's not valid, then server will try to get invalid stream source.